README.md

```diff
@@ -1,2 +1,7 @@
 # Hello-Microverse-Project
 Setup of Hello world using HTML and CSS linters and a test.md
+#Added an index.html file to the project.
+#Added a style.css file to the project.
+#Created test.md file and added it to .gitignore file.
+#Added node_modules to gitignore file.
+#successfully added the linters and run it successfully.
```

Comment:
- [ ] In my opinion there are several sections of the README that are missing, let us use this [template] provided by provided by microverse :sunglasses: to make it look more professional